lib/searchyll/indexer.rb

```diff
@@ -30,11 +31,19 @@ def initialize(configuration)
       self.working       = true
       self.timestamp     = Time.now
       self.batch_size    = BATCH_SIZE
+      
+      # Compute a regex for detecting paths to ignore
+      escaped = (configuration.elasticsearch_ignore.map {|i| Regexp.escape(i).gsub('\*','.+?')}).join('|')
+      self.ignore_regex = Regexp.new "^(#{escaped})$", Regexp::IGNORECASE
```

Comment:
Let's put this in a `begin/rescue` with `Jekyll.logger.error` output so we can understand what the `elasticsearch_ignore` values are and the error in the process.